packages/dataviews/src/field-types/datetime.tsx

```diff
@@ -42,5 +58,9 @@ export default {
 	enableSorting: true,
 	filterBy: {
 		operators,
+		overrideOperatorLabels: {
```

Comment:
Instead of creating a separate key to override the label, what if we make `filterBy.operators` take either a string (key) or the operator definition (key, label)?

Reply:
That was the initial implementation. However, we decided to go with the current implementation because we don't think customers would need to override the operators.

https://github.com/Automattic/wp-calypso/pull/103660#issuecomment-2904682558